src/VisualStudio/CSharp/Test/EditorConfigSettings/DataProvider/DataProviderTests.cs

```diff
@@ -165,9 +165,10 @@ public void TestGettingCodeStyleSettingsProviderLanguageServiceAsync()
             settingsProvider.RegisterViewModel(model);
             var dataSnapShot = settingsProvider.GetCurrentDataSnapshot();
 
-            // We don't support PreferredModifierOrder yet:
+            // We don't support PreferredModifierOrder and PreferStaticAnonymousFunction yet:
             var optionsWithUI = CSharpCodeStyleOptions.AllOptions
-                .Remove(CSharpCodeStyleOptions.PreferredModifierOrder);
+                .Remove(CSharpCodeStyleOptions.PreferredModifierOrder)
+                .Remove(CSharpCodeStyleOptions.PreferStaticAnonymousFunction);
```

Comment:
I guess this is not a good solution and I will have to add editorconfig UI support for the new option, right? Would like a hint where to look to add such support then)

Reply:
yeah, you shouldn't need this.  see whereever we handled PreferStaticLocalFucntoin and update all those locatoins as well.